R/getOOBPreds.R

```diff
@@ -64,6 +64,7 @@ getOOBPredsLearner.BaseWrapper = function(.learner, .model) {
 # checks if the model was trained on the corresponding task by comparing
 # the descriptions
 checkModelCorrespondsTask = function(model, task) {
-  if (!identical(task$task.desc, model$task.desc))
+  compare = c("id", "type", "target", "n.feats", "has.weights", "has.blocking", "is.spatial", "positive")
```

Comment:
Is there any way to get this programmatically?

Reply:
I'd suggest to write a hash function for task descriptions and save the hashes in the task descriptions. This is cleaner and eventually even cheaper than the bitwise comparison done here. Must not be in this PR though.